sunpy/coordinates/frames.py

```diff
@@ -605,6 +605,24 @@ def make_3d(self):
                                                           lat=lat,
                                                           distance=d))
 
+    def is_visible(self):
+        """
+        Returns whether the coordinate is on the visible side of the Sun.
```

Comment:
Side is probably not the best phrasing given the nuance with the distance we discussed earlier?

Reply:
Also might be worth adding an example of realise frame to change rsun in this docstring?

Reply:
yeah, it would be nice to explain what this could mean (i.e. that it could be on the other side), and checks that its visible from the observer in the HPC frame.

Reply:
> Also might be worth adding an example of realise frame to change rsun in this docstring?

Definitely not

Reply:
or at least linking out to it, because people wont know it exists.

Reply:
I'm not a fan of adding a reference to `realize_frame()`.

Reply:
"side" is further clarified lower down in the docstring so I think the wording here is ok.